assets/js/googlesitekit/datastore/user/dismissed-items.js

```diff
@@ -134,6 +134,9 @@ const baseSelectors = {
 	 * @return {(boolean|undefined)} TRUE if dismissed, otherwise FALSE, `undefined` if not resolved yet.
 	 */
 	isItemDismissed: createRegistrySelector( ( select ) => ( state, slug ) => {
+		if ( slug === '' ) {
+			return undefined;
+		}
```

Comment:
This change is not needed. You can just omit calling the selector if the `dismiss` property is empty:

```js
const isDismissed = useSelect( ( select ) =>
	dismiss ? select( CORE_USER ).isItemDismissed( dismiss ) : undefined
);
```

Please, revert this change and the one in the tests file.